nilearn/tests/test_signal.py

```diff
@@ -671,17 +739,26 @@ def test_clean_confounds_errros(signals):
     ):
         clean(signals, ensure_finite=None)
 
+    # test boolean is not given to signal.clean
+    with pytest.raises(TypeError, match="high/low pass must be float or None"):
+        clean(signals, low_pass=False)
+
+    with pytest.raises(TypeError, match="high/low pass must be float or None"):
+        clean(signals, high_pass=False)
+
 
 def test_clean_confounds():
+    """Check output of cleaning when counfoun is passed."""
     signals, noises, confounds = generate_signals(
         n_features=41, n_confounds=5, length=45
     )
     # No signal: output must be zero.
-    eps = np.finfo(np.float64).eps
+    eps = EPS
```

Comment:
Is there a reason why you don't want to use the global variable here inside the tests?

Reply:
nope: that's a search and replace issue
let me fix